### ISSUE TITLE
Add Habitat Availability and Use Data

### ISSUE THREAD

**Issue by mackerman44** [mackerman44/telemetyr#24]
Add the available habitat availability and use data from 2018/2019 season. The habitat availability data includes:

- OTG habitat transect data

- Drone imagery (organize and store on NAS?)

- Depth and velocity rasters for a winter (and summer?) discharge scenario. These may already be available in the mra_hsi repo.

- HOBO temp data

- Other?

Then start the scripts/functions to roll up the availability data and save to appropriate locations. Also need to store the habitat use data from Nick, Tulley, Mike H., likely to the NAS.

**Comment by KevinSee** [comment]
I prepped the OTG habitat availability transect data, and saved it on the NAS, in _data/habitat/lemhi_telemetry/prepped/habitat_available.rda_.

I also prepped the OTG habitat use data, trying to make it match the availability as much as possible (names of various factor levels, etc.). That is also saved on the NAS, in _data/habitat/lemhi_telemetry/prepped/habitat_use.rda_.

The depth and velocity raster values at OTG sampled transects have been prepped by Richie. They live on the NAS, in _data/habitat/lemhi_telemetry/availability/cross_sections/DV_extract_ and are named by sinuosity class (Low, Med, High) followed by "d" or "v" for depth or velocity. Still waiting on the entire raster to be parsed by sinuosity class.
